### PR TITLE
Pan/Improved codes based on Long's comments

### PR DESCRIPTION
**Pull request by ttcode10**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,8 +22,6 @@ const App = () => {
           <Route path="/tasks" component={BrowseTask} />
           <Route exact path="/create-profile" component={CreateProfile} />
           <Route exact path="/create" component={PostTask} />
-          <Route exact path="/signup" component={Signup} />
-          <Route exact path="/login" component={Login} />
```
Comment (ttcode10) [inline src/App.js:28]: 此处有疑问，因signup/login现在是弹窗，是不是就不用配置路由？

**Comment by KieraDOG** [comment]
嗯 具体看你们的设计 理论上讲就不需要了

________________________________
发件人: ttcode10 <notifications@github.com>
发送时间: Wednesday, September 9, 2020 1:06:02 AM
收件人: cb-paradise-project-frontend/eureka-v1.1.0 <eureka-v1.1.0@noreply.github.com>
抄送: Long Zhao <zlong@outlook.com>; Review requested <review_requested@noreply.github.com>
主题: Re: [cb-paradise-project-frontend/eureka-v1.1.0] Pan/Improved codes based on Long's comments (#13)


@ttcode10 commented on this pull request.

________________________________

In src/App.js<https://github.com/cb-paradise-project-frontend/eureka-v1.1.0/pull/13#discussion_r484994236>:

> -          <Route exact path="/signup" component={Signup} />

-          <Route exact path="/login" component={Login} />



此处有疑问，因signup/login现在是弹窗，是不是就不用配置路由？

―

File: src/pages/Landing/Navigation/Navigation.js [hunk 2]
```diff
@@ -31,17 +31,17 @@ function Navigation() {
...
-                <Link onClick={showLoginModal} className = {styles.logIn}>Log in</Link>
-                <Link onClick={showSignupModal} className = {styles.signUp}>Sign up</Link>
+                <button onClick={showLoginModal} className = {styles.logIn}>Log in</button>
+                <button onClick={showSignupModal} className = {styles.signUp}>Sign up</button>
               </div>
               <>
                 {
-                  !!(authModalVisible.loginModal) ? <Login /> : null
+                  !!(authModalVisible.loginModal) && <LoginModal />
                 }
```
Comment (KieraDOG) [inline src/pages/Landing/Navigation/Navigation.js:40]: 这里也不推荐换行，推荐去读 airbnb react style

File: src/pages/Landing/Navigation/Navigation.module.scss [hunk 3]
```diff
@@ -154,6 +154,11 @@ nav {
       background-color: #fff;
     }
 
+    button {
+      background-color: transparent;
+      border: none;
+    }
+
```
Comment (KieraDOG) [inline src/pages/Landing/Navigation/Navigation.module.scss:161]: css module 写法应该是每个 css 都是 class selector

**Review by KieraDOG — APPROVED** [review]
LGTM 👍

File: src/pages/Landing/Navigation/Navigation.module.scss (hunk 3 above)
Comment (JavaScriptN0ob) [reply src/pages/Landing/Navigation/Navigation.module.scss:161]: .button_className {
     background-color: transparent;
     border: none;
}

File: src/pages/Landing/Navigation/Navigation.js (hunk 2 above)
Comment (JavaScriptN0ob) [reply src/pages/Landing/Navigation/Navigation.js:40]: https://github.com/airbnb/javascript/tree/master/react

**Comment by JavaScriptN0ob** [comment]
LGTM too

**Review by JavaScriptN0ob — APPROVED** [review]

**Review by JinNGuo — APPROVED** [review]
LGTM. :P

**Review by hachilam92 — APPROVED** [review]

**Pull request merged**